index.html

```diff
@@ -18,9 +18,11 @@
         </button>
         <div class="collapse navbar-collapse" id="navbarText">
             <ul class="navbar-nav mr-auto">
-                <a class="nav-link" href="https://github.com/terrajobst/sitesof.net">
+                <li class="nav-item">
+                    <a class="nav-link" href="https://github.com/terrajobst/sitesof.net">
                     <span class="oi oi-code pr-1"></span> Source
                 </a>
```

Comment:
Looks like the indentation is slightly off here

Reply:
Turns out I made a change later that conflicted, so I rebased your changes on top and resolved the merge conflict. While at it, I also fixed the indentation.